pkg/git/git.go

```diff
@@ -54,6 +54,8 @@ const (
 	DefaultBranch            = "master"
 
 	defaultGithubAuthRoot = "git@github.com:"
+	defaultGitUser        = "Anago GCB"
+	defaultGitEmail       = "nobody@k8s.io"
```

Comment:
We may want to consider changing these in future (post-implementation)

Reply:
I agree :+1: